src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/Serializer/ProtobufOtlpMetricSerializer.cs

```diff
@@ -150,7 +150,9 @@ private static int WriteMetric(byte[] buffer, int writePosition, Metric metric)
                     writePosition += ReserveSizeForLength;
 
                     writePosition = ProtobufSerializer.WriteBoolWithTag(buffer, writePosition, ProtobufOtlpMetricFieldNumberConstants.Sum_Is_Monotonic, metric.MetricType == MetricType.LongSum);
-                    writePosition = ProtobufSerializer.WriteEnumWithTag(buffer, writePosition, ProtobufOtlpMetricFieldNumberConstants.Sum_Aggregation_Temporality, metric.Temporality == AggregationTemporality.Cumulative ? 2 : 1);
+
+                    var aggregationValue = metric.Temporality == AggregationTemporality.Cumulative ? ProtobufOtlpMetricFieldNumberConstants.Aggregation_Temporality_Cumulative : ProtobufOtlpMetricFieldNumberConstants.Aggregation_Temporality_Delta;
```

Comment:
Do you know why previous PR tests did not expect this one?

Reply:
nit:

* Move this up before the `switch` so all the different types don't need to reproduce the code.
* Consider making this multi-line.

   ```diff
   -var aggregationValue = metric.Temporality == AggregationTemporality.Cumulative ? ProtobufOtlpMetricFieldNumberConstants.Aggregation_Temporality_Cumulative : ProtobufOtlpMetricFieldNumberConstants.Aggregation_Temporality_Delta;
   +var aggregationValue = metric.Temporality == AggregationTemporality.Cumulative
   +   ? ProtobufOtlpMetricFieldNumberConstants.Aggregation_Temporality_Cumulative
   +   : ProtobufOtlpMetricFieldNumberConstants.Aggregation_Temporality_Delta;
   ```

   Better readability IMO.